ui/simul/src/simul.home.ts

```diff
@@ -0,0 +1,9 @@
+import StrongSocket from 'common/socket';
+
+site.load.then(async() => {
+  site.socket = new StrongSocket(`/socket/v5`, false, { params: { flag: 'simul' } });
+  const rsp = await fetch('/simul/reload');
+  const html = await rsp.text();
+  $('.simul-list__content').html(html);
+  site.pubsub.emit('content-loaded');
```

Comment:
Looks like this will reload simuls from XHR once, on page load.

Instead we we want to reload from XHR every time there's new data, as notified by the `socket.in.reload` event. And not on page load. The initial page already contains the simuls, we wouldn't do an XHR just after page load.

Reply:
I didn't understand it either, but I thought it was functionally equivalent to what was previously embedded in server.

Reply:
Was the behavior different before?

Reply:
oh, i missed the socket.in.reload bit. good catch